dashboard/src/components/partials/ChecksList/CheckPublishInfo.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import CronDescriptor from "/components/partials/CronDescriptor";
+
+class CheckPublishInfo extends React.Component {
+  static propTypes = {
+    check: PropTypes.object.isRequired,
+  };
+
+  render() {
+    const { check } = this.props;
+
+    return (
+      <React.Fragment>
+        Executed{" "}
+        <strong>
+          {check.interval ? (
```

Comment:
Ideally this component would include a query fragment describing the fields it requires. In this case it's referencing the `interval`, `cron`, and `subscription` fields.